app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def self.update_posts_counter(author_id)
+    author = User.find(author_id)
+    author.posts.increment!(:comments_counter)
```

Comment:
- [Optional] This line should increment post_counter of the author.
`author.increment!(:post_counter)`